includes/CLI/Plugin_Check_Command.php

```diff
@@ -111,6 +111,12 @@ public function __construct( Plugin_Context $plugin_context ) {
 	 * [--warning-severity=<warning-severity>]
 	 * : Warning severity level.
 	 *
+	 * [--include-low-severity-errors]
+	 * : Include errors with lower severity than the threshold as other type.
+	 *
+	 * [--include-low-severity-warnings]
+	 * : Include errors with lower severity than the threshold as other type.
```

Comment:
```suggestion
	 * : Include warnings with lower severity than the threshold as other type.
```

Reply:
@davidperezgar This is not fixed in the PR.

Reply:
you're right.